HelpSource/Tutorials/Getting-Started/04-Functions-and-Other-Functionality.schelp

```diff
@@ -3,7 +3,11 @@ summary:: Getting Started With SuperCollider
 categories:: Tutorials>Getting-Started
 related:: Tutorials/Getting-Started/00-Getting-Started-With-SC
 
-The easiest way to get sound from SC is to use a Function. Below is a simple example of this. Execute this (after making sure the server is booted), and when you're sick of it, press Cmd - . (that's hold down the command key and press the period or fullstop key) to stop the sound. This will always stop all sound in SC. You'll be using it a lot, so commit it to memory.
+Warning! If you are using headphones or external speakers, it is recommended to set your volume low before playing the following example and then bring it up to a comfortable level. Raw synthesized tones can sometimes seem louder than music and the other forms of audio we usually listen to.
```

Comment:
There is a label in the `SCDoc`-syntax that is meant for warinings. You can write:

```
 WARNING:: If you are …

```